susemanager-branding-oss/susemanager-branding-oss.spec

```diff
@@ -16,30 +16,29 @@
 #
 
 
-%global susemanager_shared_path /usr/share/susemanager
+%global susemanager_shared_path %{_datadir}/susemanager
 %global wwwroot %{susemanager_shared_path}/www
 %global wwwdocroot %{wwwroot}/htdocs
-
 Name:           susemanager-branding-oss
-Version:        5.0.4
+Version:        5.1.0
 Release:        0
 Summary:        SUSE Manager branding oss specific files
 License:        GPL-2.0-only
+# FIXME: use correct group or remove it, see "https://en.opensuse.org/openSUSE:Package_group_guidelines"
 Group:          Applications/System
 URL:            https://github.com/uyuni-project/uyuni
 Source0:        %{name}-%{version}.tar.gz
-BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+BuildRequires:  skelcd-EULA-suse-manager-server-container
+Requires:       skelcd-EULA-suse-manager-server-container
+Conflicts:      oracle-server
```

Comment:
At some point we could do a bit more of clean to remove the oracle traces. But not today.